pkg/composableschemadsl/compiler/node.go

```diff
@@ -35,6 +35,15 @@ func (tn *dslNode) Connect(predicate string, other parser.AstNode) {
 	tn.children[predicate].PushBack(other)
 }
 
+// Used to preserve import order when doing import operations on AST
+func (tn *dslNode) ConnectAndHoistMany(predicate string, other *list.List) {
```

Comment:
does the order really matter?

Reply:
It doesn't, but it means that tests pass without regeneration (which makes me feel better about not having missed something) and that the generated output matches my mental model of imports (in terms of imports being replaced by the contents of the referenced files).

Reply:
Okay. Maybe name "ConnectAtFront"?

Reply:
I'd be more keen to unwind it in a later refactor if we want to drop it as an optimization

Reply:
ok